src/toast/redux/toast.spec.js

```diff
@@ -29,7 +29,11 @@ describe('toast/redux', () => {
       expect(reducer(state, { type: 'UNKNOWN' }))
         .toBe(state)
     )
-    it('should create a toast', () =>
+    it('should create a toast with type option', () =>
+      expect(reducer(state.ui.toast, create('success toast', { type: 'success' })))
```

Comment:
- is default `success` ?
 - if the answer is `yes` : add an other type in this test

Reply:
by default, type is empty string, 
```es6 
Toast.defaultProps = {
...
  type: '',
}
```

Reply:
I speak about the type in `create({ title: 'Default' })`

Reply:
`create` action, by default `type` === `undefined`
```es6
export const create = (title, options) => action(/* type */)(title, options)
```
Or I don't understand your question ... :/

Reply:
When I do a : `create({ title: 'yo' })` there is not default `type` field set.
1. Can you add a default `type` field when none is given ?
2. If the default type value is `default` can you change your test to make sure this is overridden  ? 

Thanks